### ISSUE TITLE
Statistics > Other: Call to undefined method ExtendedFamilyTranslations::danishTranslations

### ISSUE THREAD

**Issue by TheDutchJewel** [hartenthaler/hh_extended_family#140]
No matter which language I use, always get this error only in the Statistics > Other tab:

**Call to undefined method Hartenthaler\Webtrees\Module\ExtendedFamily\ExtendedFamilyTranslations::danishTranslations() …\modules_v4\hh_extended_family\ExtendedFamilyTabModule.php:577**

![afbeelding](https://user-images.githubusercontent.com/46184729/152410072-d09c04cb-7601-4d97-a638-d4ba7b9d6b62.png)

Tested with webtrees v2.0.19 and all modules disabled except 'hh_extended_family' (Extended Family v2.1.0.1).

**Comment by hartenthaler** [comment]
Strange! The "StatisticsChartModule" calls ... "commonCountriesList" calls ... "PlaceRepository" for Danish Language. Maybe ok so far. But why is this calling the translation function in the "ExtendedFamilyModule"? There is no function "danishTranslations" in my module. The lines around the mentioned line #577 in \modules_v4\hh_extended_family\ExtendedFamilyTabModule.php:577 are used to call the germanTranslations. Are you sure that you are using the original file "ExtendedFamilyTabModule.php"?

**Comment by TheDutchJewel** [comment]
No, apparently it was not the original version of" ExtendedFamilyTranslations.php".

I did use the latest version of the module, but there were still language sections in "ExtendedFamilyTranslations.php" that had since been removed in "ExtendedFamilyTabModule.php". One of them was the Danish section.

After removing those sections in "ExtendedFamilyTranslations.php" (or using the original file) the problem was solved.

**Comment by TheDutchJewel** [comment]
No, apparently it was not the original version of" ExtendedFamilyTranslations.php".

It seems I used an updated version from the master branch. In it, certain translation sections (including Danish) are missing, while according to the "ExtendedFamilyTabModule.php" file, they were expected to be present.

After adding the missing parts in "ExtendedFamilyTranslations.php" (or using the original file) the problem was solved.
